src/app/core/interceptors/httpInterceptor.ts

```
@@ -168,7 +168,7 @@ export class HttpConfigInterceptor implements HttpInterceptor {
                 this.secureStorageService.clearAll();
                 this.storageService.clearAll();
                 globalCacheBusterNotifier.next();
-                return EMPTY;
+                return throwError(error);
```

this was changed recently in my prev PR: #3263 [ref](https://github.com/fylein/fyle-mobile-app/pull/3263/files#diff-073e990716b75fbf4f594d5c5ecc68c18857e9ccd69d78e52c31e51264cba5fa)
the change of swallowing error is not needed here, so reverting change for this one line